x-pack/legacy/plugins/infra/public/pages/logs/analysis/page_setup_content.tsx

```diff
@@ -4,13 +4,212 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import React from 'react';
-
+import React, { useState, useMemo } from 'react';
+import moment, { Moment } from 'moment';
+import {
+  EuiPage,
+  EuiPageBody,
+  EuiPageContent,
+  EuiPageContentHeader,
+  EuiPageContentHeaderSection,
+  EuiPageContentBody,
+  EuiText,
+  EuiTitle,
+  EuiSpacer,
+  EuiButton,
+  EuiButtonIcon,
+  EuiDescribedFormGroup,
+  EuiFormRow,
+  EuiDatePicker,
+  EuiFlexGroup,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+import euiStyled from '../../../../../../common/eui_styled_components';
 import { useTrackPageview } from '../../../hooks/use_track_metric';
 
-export const AnalysisSetupContent = () => {
+interface AnalysisSetupContentProps {
+  setupMlModule: (startTime: number | undefined, endTime: number | undefined) => Promise<any>;
+  isSettingUp: boolean;
+}
+
+const startTimeLabel = i18n.translate('xpack.infra.analysisSetup.startTimeLabel', {
+  defaultMessage: 'Start time',
+});
+const endTimeLabel = i18n.translate('xpack.infra.analysisSetup.endTimeLabel', {
+  defaultMessage: 'End time',
+});
+const startTimeDefaultDescription = i18n.translate(
+  'xpack.infra.analysisSetup.startTimeDefaultDescription',
+  {
+    defaultMessage: 'now',
```

Comment:
```suggestion
    defaultMessage: 'Start of log indices',
```

Reply:
I was wondering about this. I almost want this to drop the start/end inputs completely by default to make the most common experience less noisy...

<img width="707" alt="Screen Shot 2019-08-16 at 1 56 29 PM" src="https://user-images.githubusercontent.com/159370/63187803-c1dc1c80-c02d-11e9-899b-9a99ebcaf75f.png">

Then if you choose to configure, it reveals the controls:

<img width="821" alt="Screen Shot 2019-08-16 at 1 50 48 PM" src="https://user-images.githubusercontent.com/159370/63187832-d0c2cf00-c02d-11e9-9c5d-6326ea43f885.png">

(Probably don't even need a way to exit out of those controls since the defaults there are the same as when you aren't showing the controls.)

Thoughts?

Reply:
I like this - it does remove a lot of the excess in the common use case.